dev/io.openliberty.com.sun.xml.messaging.saaj/saaj.2.0.bnd

```diff
@@ -41,4 +54,4 @@ Service-Component: \
     implementation:=com.ibm.wsspi.classloading.ResourceProvider; \
```

Comment:
Should the Service-Component name be updated to io.openliberty.com.sun.xml.messaging.saaj.2.0.ResourceProvider to make it unique?